src/services/posts/fetch-posts.js

```diff
@@ -31,13 +32,15 @@ const fetchPosts = async({isFlagged, cursor, limit }) => {
 	const whereClause = (whereClauseArray.length > 0 ? ' WHERE ' : '') 
 	+ whereClauseArray.join(' AND ')
 
-	const feed = await db.query(`
+	const result = await db.query(`
 ${selectClause} ${fromClause} ${joinClause} 
 ${whereClause} ${orderByClause} ${limitClause};
 `
 	) 
-	
-	return feed.rows
+	const feed = result.rows
+	const value = await db.query(` SELECT COUNT(posts) FROM posts;`)
```

Comment:
the count should factor in the isFlagged query. 

When the client is fetching all flagged posts, the total count in that case should be the total number of flagged posts

Do remember that, the total count is to help the frontend know whether to make a call for the next set of data or not. So if the number of flagged posts is 3, and the totalCount is 15. The frontend/client will keep making calls thinking there are 15 flagged posts

Reply:
this is awesome !